### PR TITLE
[External_API doc] Added interactions API doc

### PR DESCRIPTION
**Pull request by jiepiao**
## Description
Added reminders external API doc and webhook event

#### Please review:
1. Interactions section
2. Changelog

### CODE REVIEW

File: source/includes/_interactions.md [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+# Meeting/Call type
+{
+    "date": "2022-02-22T11:15:29.758-08:00",
+    "id": 3007,
+    "attendees": [
+        "john@affinity.co",
+        "yen@alice.com"
+    ],
+    "start_time": "2022-02-22T11:15:29.758-08:00",
+    "end_time": null,
```
Comment (dachan) [inline source/includes/_interactions.md:22]: Is end time only relevant when meeting is automatically logged? Is start time always the same as date? When you create a meeting or call by the external API does it set the start time equal to date as well?

File: source/includes/_interactions.md [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+            ],
+            "cc": [],
+            "direction": 0
+        }
+    ],
+    "next_page_token": "eyJwYXJhbXMiOnsiY29tcGxldGVyX2lkIjpudWxsLCJvd25lcl9pZCI6bnVsbCwiY3JlYXRvcl9"
+}
+```
+
+`GET /interactions`
```
Comment (dachan) [inline source/includes/_interactions.md:187]: Did we drop logging_type?

File: source/includes/_interactions.md (hunk 1 above)
Comment (jiepiao) [reply source/includes/_interactions.md:22]: Yes, the date is always same as `start_time`. It is a little confusing, do we want to remove the date field from response?

File: source/includes/_interactions.md (hunk 2 above)
Comment (jiepiao) [reply source/includes/_interactions.md:187]: Yes, I will update the PR shortly.

**Comment by doctorzippy** [comment]
I reviewed with David and made changes. Also added some language with Jie clarifying that the endpoint is in beta.

**Comment by doctorzippy** [comment]
Looks good to me

**Review by doctorzippy — APPROVED** [review]
Looks good to me based on the last set of changes I walked through with David and Jie.

**Pull request merged**